### PR TITLE
Core/Spells: Implemented monk stagger

### PR DESCRIPTION
**Pull request by matanshukry**
**Changes proposed:**

-  Implemented monk stagger
- 
**Target branch(es):** 3.3.5/master

- [ ] 3.3.5
- [x] master

**Issues addressed:**

**Tests performed:**

- [x] It builds
- [x] Tested in-game

**Known issues and TODO list:** (add/remove lines as needed)

### CODE REVIEW

File: src/server/scripts/Spells/spell_monk.cpp [hunk 1]
```diff
@@ -159,9 +164,233 @@ class spell_monk_provoke : public SpellScript
     }
 };
 
+// Utiliy for stagger scripts
```
Comment (Faq) [inline src/server/scripts/Spells/spell_monk.cpp:167]:
```suggestion
// Utility for stagger scripts
```

File: sql/updates/world/master/2021_99_99_99_world_monk_stagger.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+-- Stagger
+
```
Comment (Faq) [inline sql/updates/world/master/2021_99_99_99_world_monk_stagger.sql:2]:
```suggestion
```

Comment (matanshukry) [reply sql/updates/world/master/2021_99_99_99_world_monk_stagger.sql:2]: I like this empty line :p do I have to remove it?

Comment (illfated) [reply sql/updates/world/master/2021_99_99_99_world_monk_stagger.sql:2]: Cosmetic, not very important. We're just used to seeing the first comment line directly on top of the following code, at least when there is a complete `DELETE FROM / INSERT INTO` query.

Comment (matanshukry) [reply sql/updates/world/master/2021_99_99_99_world_monk_stagger.sql:2]: mah you bought me with "used to seeing" and "codestyle" - I can't resist a standard.
removed the empty line.

**Pull request merged**